WordPress/openverse-frontend
# Remove Vocabulary icon font

## Description


This PR removes deprecated Vocabulary icon font.

The external link icon in two places was still using the icon font. I replaced them with VIcon with an svg icon. If the sizes need updating, that should be done in a separate PR:
<img width="937" alt="Screen Shot 2022-03-10 at 2 14 25 PM" src="https://user-images.githubusercontent.com/15233243/157652945-35ad612f-c901-447d-9cad-79ea82046148.png">
<img width="345" alt="Screen Shot 2022-03-10 at 2 12 34 PM" src="https://user-images.githubusercontent.com/15233243/157652952-57bf681e-ba8b-49c7-bd80-304e0d68b07b.png">


Two components that are not used anymore were removed:
- FilterTag
- Dropdown

## Testing Instructions

Check the meta search form and the sources page, the external link icons should be displayed correctly.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
